File: src/Z.Core/System.Decimal/System.Decimal/Decimal.Subtract.cs
// Description: C# Extension Methods Library to enhances the .NET Framework by adding hundreds of new methods. It drastically increases developers productivity and code readability. Support C# and VB.NET
// Website & Documentation: https://github.com/zzzprojects/Z.ExtensionMethods
// Forum: https://github.com/zzzprojects/Z.ExtensionMethods/issues
// License: https://github.com/zzzprojects/Z.ExtensionMethods/blob/master/LICENSE
// More projects: http://www.zzzprojects.com/
// Copyright © ZZZ Projects Inc. 2014 - 2016. All rights reserved.
using System;

public static partial class Extensions
{
    /// <summary>
    ///     Subtracts one specified  value from another.
    /// </summary>
    /// <param name="d1">The minuend.</param>
    /// <param name="d2">The subtrahend.</param>
    /// <returns>The result of subtracting  from .</returns>
    public static Decimal Subtract(this Decimal d1, Decimal d2)
    {
        return Decimal.Subtract(d1, d2);
    }
}

File: src (VB.NET)/Z.ExtensionMethods.VB/Z.Core/System.Decimal/System.Decimal/Decimal.Subtract.vb

' Copyright (c) 2015 ZZZ Projects. All rights reserved
' Licensed under MIT License (MIT) (https://github.com/zzzprojects/Z.ExtensionMethods)
' Website: http://www.zzzprojects.com/
' Feedback / Feature Requests / Issues : http://zzzprojects.uservoice.com/forums/283927
' All ZZZ Projects products: Entity Framework Extensions / Bulk Operations / Extension Methods /Icon Library


Public Module Extensions_172

	''' <summary>
	'''     Subtracts one specified  value from another.
	''' </summary>
	''' <param name="d1">The minuend.</param>
	''' <param name="d2">The subtrahend.</param>
	''' <returns>The result of subtracting  from .</returns>
	<System.Runtime.CompilerServices.Extension> _
	Public Function Subtract(d1 As [Decimal], d2 As [Decimal]) As [Decimal]
		Return [Decimal].Subtract(d1, d2)
	End Function
End Module


